exercises/meetup/canonical-data.json

```diff
@@ -3,446 +3,567 @@
-      "property": "meetup",
-      "year": 2013,
-      "month": 4,
-      "week": "second",
-      "dayofweek": "Monday",
-      "dayofmonth": 8
-    },
-    {
-      "description": "second Tuesday of May 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 5,
-      "week": "second",
-      "dayofweek": "Tuesday",
-      "dayofmonth": 14
-    },
-    {
-      "description": "second Tuesday of June 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 6,
-      "week": "second",
-      "dayofweek": "Tuesday",
-      "dayofmonth": 11
-    },
-    {
-      "description": "second Wednesday of July 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 7,
-      "week": "second",
-      "dayofweek": "Wednesday",
-      "dayofmonth": 10
-    },
-    {
-      "description": "second Wednesday of August 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 8,
-      "week": "second",
-      "dayofweek": "Wednesday",
-      "dayofmonth": 14
-    },
-    {
-      "description": "second Thursday of September 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 9,
-      "week": "second",
-      "dayofweek": "Thursday",
-      "dayofmonth": 12
-    },
-    {
-      "description": "second Thursday of October 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 10,
-      "week": "second",
-      "dayofweek": "Thursday",
-      "dayofmonth": 10
-    },
-    {
-      "description": "second Friday of November 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 11,
-      "week": "second",
-      "dayofweek": "Friday",
-      "dayofmonth": 8
-    },
-    {
-      "description": "second Friday of December 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 12,
-      "week": "second",
-      "dayofweek": "Friday",
-      "dayofmonth": 13
-    },
-    {
-      "description": "second Saturday of January 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 1,
-      "week": "second",
-      "dayofweek": "Saturday",
-      "dayofmonth": 12
-    },
-    {
-      "description": "second Saturday of February 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 2,
-      "week": "second",
-      "dayofweek": "Saturday",
-      "dayofmonth": 9
-    },
-    {
-      "description": "second Sunday of March 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 3,
-      "week": "second",
-      "dayofweek": "Sunday",
-      "dayofmonth": 10
-    },
-    {
-      "description": "second Sunday of April 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 4,
-      "week": "second",
-      "dayofweek": "Sunday",
-      "dayofmonth": 14
-    },
+      "description": "test -teenth",
+      "cases":[
+        {
+          "description": "monteenth",
+          "cases": [
+            {
+              "description": "testcase 1 : May 2013",
```

Comment:
> testcase 1 : May 2013

What is this describing?
That it is "testcase 1" can be determined by the array ordering, so is unnecessary in the description.
Why are we testing May and not June? Why are we testing 2013 and not 2014?